README.md

```diff
@@ -68,14 +84,69 @@ const playbackId = await Video.Assets.createPlaybackId(asset.id, {
 
 Or, if you don't have the files online already, you can ingest one via the direct uploads API.
 
+However, before that, a word of warning on using request libraries.
+
+### Request Library Usage in Node
+
+Since 2019, `request` has gone into maintenance mode. 
+
+We must use an alternate request library such as `node-fetch` [here](https://github.com/node-fetch/node-fetch/).
+
+**With CommonJS**
+
+Install version 2 of `node-fetch`
```

Comment:
does CJS _only_ support v2 of node-fetch? Or rather, does v2 only support CJS? Thinking if it's more helpful to phrase this as "For v2 of node-fetch, do this: ... For v3, do this", but I might be a little confused here. I hate javascript dependency management of ES6/CJS/etc.

Reply:
`node-fetch` says that its v3 is an ESM-only module but I suppose for versions below you can use either CJS or ESM. So It's more like, "If for whatever reason, you _have_ to use v3 of of `node-fetch`, you supposedly have to use ESM only because it's an ESM-only module." Apparently v3 of `node-fetch` cannot be imported with `require`.

People _can_ also use something other than `node-fetch`.

Reply:
What about if we phrase this simply as, part of the above comment, "Instead, we're using node-fetch (latest/v3 - not to be said here, but kind of assumed, and which then only lets us use ESM), so here's how it's done". A node-fetch v2 example can be helpful, but I'm not sure that putting them together in the same place is helping more than it might be confusing people.

We could also put a full example somewhere else that's node-fetch v2 + CJS in another file somewhere, if we want to have that example there, too.